tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/AnalyticSummaryPopup.xaml

```diff
@@ -0,0 +1,125 @@
+<!-- Copyright (c) Microsoft Corporation.. -->
+<!-- Licensed under the MIT License. -->
+
+<ContentDialog
+    x:Class="DevHome.QuietBackgroundProcesses.UI.Views.AnalyticSummaryPopup"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.QuietBackgroundProcesses.UI.Views"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    Style="{StaticResource DefaultContentDialogStyle}">
+
+    <!-- ContentDialog Width and Height are not properly hooked up and must be set this way -->
+    <ContentDialog.Resources>
+        <x:Double x:Key="ContentDialogMinWidth">600</x:Double>
+        <x:Double x:Key="ContentDialogMaxWidth">600</x:Double>
+        <x:Double x:Key="ContentDialogMaxHeight">590</x:Double>
+        <Thickness x:Key="ContentDialogTitleMargin">0,0,0,0</Thickness>
+        <Thickness x:Key="ContentDialogPadding">0,0,0,0</Thickness>
+        <Thickness x:Key="NavigationViewContentMargin">0,0,0,0</Thickness>
+        <Thickness x:Key="SmallPinButtonMargin">0,20</Thickness>
+        <Thickness x:Key="LargePinButtonMargin">0,42</Thickness>
+        <Thickness x:Key="SmallWidgetPreviewTopMargin">0,20,0,0</Thickness>
+        <Thickness x:Key="LargeWidgetPreviewTopMargin">0,42,0,0</Thickness>
```

Comment:
These are specific to the AddWidget dialog, you don't have a PinButton so you don't need them.